src/adapter/src/coord/ddl.rs

```diff
@@ -786,11 +758,6 @@ impl Coordinator {
                 .entry(sink.cluster_id)
                 .or_default()
                 .push(sink.global_id);
-
-            // Mark the sink as dropped so we don't try to drop it again.
-            if let Some(sink) = self.active_subscribes.get_mut(&sink.global_id) {
-                sink.dropping = true;
-            }
         }
         let mut compute = self.controller.active_compute();
         for (cluster_id, ids) in by_cluster {
```

Comment:
...in the same situation as described in the last comment, we'll never call `drop_collections` on said subscribe. Is that a problem, or is the controller smart enough to clean up its internal state when it observes a `SubscribeResponse::Dropped`?

Reply:
https://github.com/MaterializeInc/materialize/pull/25179#discussion_r1487243233